docs/configuration.md

```diff
@@ -313,3 +314,24 @@ metadata:
 spec:
   defaultBrokerClass: MTChannelBasedBroker
 ```
+
+## spec.defaultChannelTemplate
+
+Knative Eventing allows you to define a default channel template when
+the user creates a generic `Channel`.
+Defaults to `messaging.knative.dev/v1beta1 InMemoryChannel` if not set.
+
+Following configuration sets default channel to `KafkaChannel` which is
+distributed with https://github.com/knative/eventing-contrib.
+
+```
+apiVersion: operator.knative.dev/v1alpha1
+kind: KnativeEventing
+metadata:
+  name: knative-eventing
+  namespace: knative-eventing
+spec:
+  defaultChannelTemplate:
```

Comment:
that would be cluster scoped?

how adding for a given set of namespaces ?   Coming to this via https://github.com/knative/docs/pull/2597/files

Reply:
Yeah, only cluster scope, similar to `defaultBrokerClass`.
We can also add support for namespace defaults, but in another PR.